src/app/personside/visittkort/body/__snapshots__/VisittkortBody.test.tsx.snap

```diff
@@ -878,15 +878,43 @@ exports[`viser info om bruker i visittkortbody 1`] = `
       </div>
     </div>
     <div
-      className="sc-kjoXOD iCDFNe"
+      className="sc-gzVnrw ljOQMZ"
     >
-      <a
-        className="lenke"
-        href="/modiapersonoversikt/brukerprofil/10108000398"
-        onClick={[Function]}
+      <svg
+        aria-label="Laster innhold"
+        className="spinner spinner--l"
+        focusable="false"
+        height={32}
+        kind="spinner"
+        onClick={undefined}
+        preserveAspectRatio="xMidYMid"
+        style={undefined}
+        viewBox="0 0 100 100"
+        width={32}
       >
-        Administrer brukerprofil
-      </a>
+        <title>
+          Venter...
```

Comment:
Her ser det ut som en reducer ikke har fått dataen sin.. Da blir ikke testen så effektiv ;-)

Reply:
I setupTests.ts settes det opp en mock-store som brukes i testene. Denne må fylles med data fra nye reducere for at testene skal rendre den endelige siden og ikke en spinner. Kan hjelpe deg med å sette opp hvis du vil :-)

Reply:
good catch, will fix